.github/workflows/pull_request.yml

```diff
@@ -19,6 +19,7 @@ on:
     branches:
       # on pull requests to master and release branches
       - master
```

Comment:
Do we need to still have the master here?

Reply:
was planning on removing it later. but may be better to remove now. (done)